verticapy/learn/ensemble.py

```diff
@@ -416,6 +416,54 @@ def get_prior(self):
         return prior_
 
 
+# ---#
+class IsolationForest(Clustering, Tree):
+    """
+---------------------------------------------------------------------------
+Creates an IsolationForest object using the Vertica IFOREST algorithm.
+
+Parameters
+----------
+name: str
+    Name of the the model. The model is stored in the DB.
+n_estimators: int, optional
+    The number of trees in the forest, an integer between 0 and 1000, inclusive.
```

Comment:
Only the upper bound is inclusive.
Valid range: (0, 1000]

Reply:
Corrected